docs/calculations/apy.md

```diff
@@ -0,0 +1,56 @@
+# APY Calculations
+
+This page demonstrates how to calculate outputs the shade derivative contracts 
+
+### stkd-SCRT APY
+Calculates the apy expected for the stkd-SCRT token
+```ts
+/**
+ * Will calculate APY for the stkd secret derivative contract
+ *
+ * returns a number that is the decimal form of the percent APY
+ */
+function calculateDerivativeScrtApy({
+  queryRouterContractAddress,
+  queryRouterCodeHash,
+  contractAddress,
+  codeHash,
+  lcdEndpoint,
+  chainId,
+}: {
+  queryRouterContractAddress: string,
+  queryRouterCodeHash?: string,
+  contractAddress: string,
+  codeHash: string,
+  lcdEndpoint: string,
+  chainId?: string,
+}): Promise<number> 
+```
+
+### dSHD APY
+Calculates the apy expected for the stkd-SCRT token
+```ts
+/**
+ * Calculates the dSHD expected APY by querying the staking contract
+ * TESTNET ONLY NOT READY FOR PRODUCTION
```

Comment:
Let's remove this from the docs for now as it doesn't need to be public facing until we got to prod